plugins/mergemedia.py

```diff
@@ -0,0 +1,114 @@
+"""MergeMedia"""
+#  Copyright (C) 2020 BY USERGE-X
+#  All rights reserved.
+#
+#  Author: https://github.com/midnightmadwalk [TG: @midnightmadwalk]
+
+
+import os
+import re
+import codecs
+import shutil
+
+from pathlib import Path
+from userge import Message, userge
+from userge.plugins.misc.upload import upload
+from userge.utils import progress, runcmd
+from hachoir.stream.input import NullStreamError, InputStreamError 
+
+
+@userge.on_cmd(
+    "mergesave",
+    about={
+        "header": "save file for {tr}merge",
+        "usage": "{tr} reply to [media] for saving it ",
+    },
+)
+async def mergesave_(message: Message):
+    """mergesave"""
+    # saving files in a separate folder.
+    r = message.reply_to_message
+    if not r:
+        await message.err("Reply To Media, dear.")
+    if not (
+      r.audio
+      or r.document
+      or r.video
+      or r.video_note
+      or r.voice
+    ):
+        await message.err("Not Supported Extension")
```

Comment:
After this need should check the mime type of document or I will just a make a function in utils for that so remove r.document or comment it out for now

Reply:
commented it out ...